enos/utils/enostask.py

```diff
@@ -70,15 +71,16 @@ def decorator(fn):
         def decorated(*args, **kwargs):
             # Constructs the environment
             kwargs['env'] = make_env(kwargs['--env'])
-            # If no directory is provided, set the default one
-            if '--env' not in kwargs:
-                kwargs['env']['resultdir'] = SYMLINK_NAME
 
             # Proceeds with the function execution
-            fn(*args, **kwargs)
+            try:
+                fn(*args, **kwargs)
+            except Exception:
+                logging.exception("Exception")
```

Comment:
The specific `logging.exception` method catches and displays the exception (including the stack trace) together with the `"Exception"` message.